Formula/a/alsa-lib.rb

```diff
@@ -1,6 +1,6 @@
 class AlsaLib < Formula
   desc "Provides audio and MIDI functionality to the Linux operating system"
-  homepage "https://www.alsa-project.org/"
+  homepage "https://www.alsa-project.org/wiki/Main_Page"
```

Comment:
It's a 301 redirect:
```
❯ curl -I "https://www.alsa-project.org/"
HTTP/2 301
server: nginx/1.22.1
date: Fri, 29 Nov 2024 19:53:06 GMT
content-type: text/html; charset=UTF-8
location: https://www.alsa-project.org/wiki/Main_Page
```

Reply:
Yes, understood, but that is longer and will break if they change the wiki for something else and don't setup the redirect, but root will likely never break. As said, not sure.

Reply:
Would it be any different if it would 301 redirect to a different gibberish url on every request, eg
* `https://www.example.org/sfsdsgertertfgOTOJ54545454TGSFFDGDFGD`
* `https://www.example.org/bofidspjfdskogsdkfhgsd45REGGFSdhdfgdg/`
* `https://www.example.org/?session=UIHUOPOKP4K543534TE3sfgdge56345gsdfdh`
...

and they would all show the same content?
Would you still want to redirect to whatever you got during the test or would you stay with the working website root?

Reply:
Focusing on your example, that's a different scenario. 301 redirecting to different URLs in short period is questionable and a different situation than a consistent 301 redirect that has been used for years (since over 15 years ago).

The former should be analyzed further on what upstream is actually doing. Latter is a more proper usage of 301.

Anyway, low priority either way so can keep.

Reply:
Yes, my example was intentionally unrealistically extreme, pointing out that we shouldn't blindly follow 301 redirects in our formulae, but add human value trough such reviews, keeping the URLs also visually nice & memorable.